WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -1058,6 +1058,12 @@ extension UIImage {
             .imageFlippedForRightToLeftLayoutDirection()
     }
 
+    /// Play square image
+    ///
+    static var playSquareImage: UIImage {
+        UIImage(systemName: "play.square")!
```

Comment:
I was looking at other systems icons and thinking maybe "creditcard" or some of it variations would fit better? Play button maybe indicates more that some video or something will be started. This icon will be hard to define since we are starting a special mode and not just showing a basic screen (normal indicator). Hm...

Reply:
I'm a bit unsure about the credit card mention because cash is also supported, I saw your comment in p91TBi-beb-p2#comment-12153 and we can wait for the final design decision. There's a separate subtask in https://github.com/woocommerce/woocommerce-ios/issues/12779 to make any design updates from the entry point p2.